chainer_chemistry/datasets/molnet/molnet.py

```diff
@@ -0,0 +1,176 @@
+from logging import getLogger
+import os
+import shutil
+
+import numpy as np
+from chainer.dataset import download
+
+from chainer_chemistry.datasets.numpy_tuple_dataset import NumpyTupleDataset
+from chainer_chemistry.dataset.parsers.csv_file_parser import CSVFileParser
+from chainer_chemistry.dataset.preprocessors.atomic_number_preprocessor import AtomicNumberPreprocessor  # NOQA
+from chainer_chemistry.datasets.molnet.molnet_config import molnet_default_config # NOQA
+
+_root = 'pfnet/chainer/molnet'
+
+
+def get_molnet_dataset(dataset_name, preprocessor=None, labels=None,
+                       split='random', frac_train=.8, frac_valid=.1,
+                       frac_test=.1, seed=777, return_smiles=False,
+                       target_index=None):
+    """Downloads, caches and preprocess MoleculeNet dataset.
+
+    Args:
+        dataset_name (str): MoleculeNet dataset name. If you want to know the
+            detail of MoleculeNet, please refer to
+            `official site <http://moleculenet.ai/datasets-1>`_
```

Comment:
When user wants to know what `dataset_name` is available for chainer_chemistry, I think user needs to see `molnet_config.py`.
So I feel it is nice to comment this fact.